django-stubs/db/backends/mysql/base.pyi

```diff
@@ -8,12 +8,15 @@ from .creation import DatabaseCreation
 from .features import DatabaseFeatures
 from .introspection import DatabaseIntrospection
 from .operations import DatabaseOperations
+from .schema import DatabaseSchemaEditor
 from .validation import DatabaseValidation
 
 version: Any
 django_conversions: Any
 server_version_re: Any
 
+class MySQLDatabase(Protocol): ...
```

Comment:
This protocol should have at least some attributes / methods

Reply:
I'm having a hard time figuring it out. I think I just need to add methods or attributes related to the connection, but is there anything else I can include?